### PR TITLE
docs(Store): add how to downgrade Store service

### PR DESCRIPTION
**Pull request by bcabanes**
Hi!

This adds a small introduction on how to use a downgraded version of the Store service in an AngularJS application.

Have a nice day!
Ben

### CODE REVIEW

File: docs/store/downgrade.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Using ngrx/store in AngularJS
+
+The _ngrx team_ has put efforts to made it possible, you can start using the
+@ngrx/store during your AngularJS to Angular conversion.
+
+## Downgrading Store service
+
+If you want to **dispatch** action or **select** some slice of your store state,
```
Comment (tdeschryver) [inline docs/store/downgrade.md]: If you want to **dispatch** __an__ action.

File: docs/store/downgrade.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+you will need to downgrade the Store service to use it in the AngularJS parts of
+you application.
```
Comment (tdeschryver) [inline docs/store/downgrade.md]: of __your__ application

File: docs/store/downgrade.md (hunk 1 above)
Comment (tdeschryver) [inline docs/store/downgrade.md]: If you want to dispatch __an__ action.

File: docs/store/downgrade.md (hunk 2 above)
Comment (tdeschryver) [inline docs/store/downgrade.md]: of __your__ application

File: docs/store/downgrade.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Using ngrx/store in AngularJS
```
Comment (brandonroberts) [inline docs/store/downgrade.md]: I think this section can be added to the beginning of the next section.

```
If you are working on an AngularJS to Angular conversion, you can use `@ngrx/store` to provide global state to your hybrid application.
```

Comment (bcabanes) [reply docs/store/downgrade.md]: Sorry, I'm not sure I totally understand what you have in mind.
Do you want to remove the second title with its text?
```
## Downgrading Store service

If you want to **dispatch** an action or **select** some slice of your store
state, you will need to downgrade the Store service to use it in the AngularJS
parts of your application.
```

Comment (brandonroberts) [reply docs/store/downgrade.md]: Replace the text below the `Using ngrx/store in AngularJS` section with the text I mentioned. Leave the Downgrade text as-is.

**Comment by brandonroberts** [comment]
Awesome! Thanks!

**Pull request merged**